### PR TITLE
API V2.3 support

### PR DESCRIPTION
**Pull request by jymnbshh**
Adding a support for API version: v2.3

### CODE REVIEW

File: src/dfs_sdk/scaffold.py [hunk 1]
```diff
@@ -21,8 +21,8 @@
 TNT = re.compile(r'datera_tenant_id\s+?=\s+?(?P<tenant_id>.*)')
 LDP = re.compile(r'datera_ldap_server\s+?=\s+?(?P<ldap>.*)')
 
-LATEST = "2.2"
-FALLBACK = ["2", "2.1"]
+LATEST = "2.3"
+FALLBACK = ["2", "2.1", "v2.2"]
```
Comment (NaveenSreeramachandra) [inline src/dfs_sdk/scaffold.py]: Can you check if the FALLBACK should include "v2.2" or "2.2" ?

**Review by NaveenSreeramachandra — COMMENTED** [review]
The rest of the code LGTM, assuming regressions are passing.

File: src/dfs_sdk/scaffold.py (hunk 1 above)
Comment (jymnbshh) [reply src/dfs_sdk/scaffold.py]: FALLBACK should be '2.2'. I've pushed the change. Please review!

**Comment by NaveenSreeramachandra** [comment]
Dumping this PR in lieu of a newer one.

**Pull request closed without merging**